textworld/generator/inform7/world2inform7.py

```diff
@@ -391,11 +393,8 @@ def generate_inform7_source(game, seed=1234, use_i7_description=False):
         remove the list of containers from L;
         remove the list of supporters from L;
         remove the list of doors from L;
-        if the number of entries in L is 1:
-            say "There is [L with indefinite articles] on the floor.";
-        else if the number of entries in L is greater than 1:
-            say "There's [L with indefinite articles] on the floor.";
```

Comment:
Not really sure why this was written like that, but maybe it's to say
```
There's two objects on the floor.
```
rather than
```
There is two objects on the floor.
```
The second one is less grammatical than the first, since people put up with more sloppiness when contractions are involved.  Ideally it would be
```
There are two objects on the floor.
```
Am I off base?

Reply:
`[L with indefinite articles]` produces something like: "a box, a chest, and a bag on the floor". I'm not sure if we can ever have something like "two <objects>" because Inform7 only group objects that have the same name, which we don't allow on our side.

Originally, the "else if" was "There are [L with ..." but you pointed out to me it was wrong (i.e. "There are a box, a chest, and a bag on the floor".